mne/gui/_coreg.py

```diff
@@ -215,10 +220,23 @@ def _set_current_fiducial(self, fid):
     def _set_info_file(self, fname):
         if fname is None:
             return
-        if not self._check_fif('info', fname):
+
+        # info file can be anything supported by read_raw
```

Comment:
On `main` it can actually be more than this -- it could be an evoked or epochs etc. -- really anything that stored an `info` in a `.fif`. This change makes it so that, for FIF, only the raw variants are supported. I'll push a change to fix this just based on a `splitext` check

Reply:
Oh never mind, I misunderstood this code -- looks like you do handle `fif` using a `read-info`-like operation below!